webdriver-spec.html

```diff
@@ -3228,7 +3228,7 @@ <h2 id=element-displayedness>Element Displayedness</h2>
   </section>
 </section>
 <section>
-  <h2>Executing Javascript</h2>
+  <h2>Executing Script</h2>
   <div class="note">
```

Comment:
Well OK, I guess, but I feel like "Executing Script" is fine since it avoids the whole JavaScript vs Javascript vs ECMAScript thing, and matches the command names.